ADOPTERS.md

```diff
@@ -3,3 +3,6 @@
 Below is a non-exhaustive list of CRI-O adopters in production environments:
 
 * [Red Hat's Openshift Container Platform](https://www.openshift.com/) uses CRI-O as the only supported CRI implementation starting with the release of OpenShift 4. CRI-O was chosen because it provides a lean, stable, simple and complete container runtime that moves in lock-step with Kubernetes, while also simplifying the support and configuration of clusters.
+
+* [Oracle Linux Cloud Native Environment](https://www.oracle.com/it-infrastructure/software.html) has used
```

Comment:
could you remove the newline please? It should correctly wrap to be fitted to screen

Reply:
I'm so used to hard wrapping Markdown files, I think this is muscle memory. Removed.

Reply:
thanks! could I bother you to squash the commits together as well?

Reply:
Squashified.

Reply:
perfection :sunglasses: